src/npm/NpmPackage.test.ts

```diff
@@ -0,0 +1,146 @@
+import * as AWS from "aws-sdk";
+import { NpmPackage } from "./NpmPackage";
+import * as sinon from "sinon";
+import * as zlib from "zlib";
```

Comment:
I could be missing something obviously, but I don't think the zlib import is needed anymore

Reply:
fixed in https://github.com/troyready/serverless-artifact-cacher/pull/3/commits/70c39de8aebb36bd15f6ae719496e8da52180cc5